src/view/SongView/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Suspense, lazy } from 'react';
+import Loader from '@/components/base/Loader';
+import { useDispatch } from 'react-redux';
+import { setCurrentSong } from '@/store/songs/SongSlice';
+import { togglePlaying } from '@/store/songs/SongSlice';
+
+const LazySongCard = lazy(() => import('@/components/custom/SongCard'));
+
+interface SongViewProps {
+    songs: any[];
+    isPlaying: boolean;
+    activeSong: any;
+   
+    handleTogglePlay: () => void;
+    
+  }
+
+  const SongView: React.FC<SongViewProps> = ({ isPlaying, activeSong,  songs, handleTogglePlay}) => {
+    
+    const dispatch = useDispatch();
```

Comment:
`const dispatch = useDispatch();`
this cannot be used in view, move it to container